src/sd-jwt/errors.ts

```diff
@@ -0,0 +1,38 @@
+/**
+ * When claims are requested but not found in the credential
+ *
+ */
+export class ClaimsNotFoundBetweenDislosures extends Error {
```

Comment:
```suggestion
export class ClaimsNotFoundBetweenDisclosures extends Error {
```

Reply:
Fixed in 3efcbac70e93f89483c6cecdb0afbb3ffd3d93ff, along with other typos.